waspc/examples/crud-testing/main.wasp

```diff
@@ -48,8 +51,8 @@ page DetailPage {
 
 entity User {=psl
   id Int @id @default(autoincrement())
-  username String @unique
-  password String
+  // username String @unique
+  // password String
```

Comment:
We can delete these?

Reply:
Just a reminder to put a :rocket: if you have done it, :+1: means just "I read it, it makes sense, will likely do it".

Reply:
I haven't done this change at the time of writing your comment, but now I have 🚀